src/components/vue-form-builder.vue

```diff
@@ -269,7 +269,14 @@ export default {
     width: 340px;
     max-width: 340px;
     border-left: 1px solid #e9edf1;
+    overflow: hidden;
+    padding-bottom: 5%;
+  }
+
+  .inspector-container > .card-body {
     overflow: auto;
+    height: 95%;
+    padding-bottom: 20px;
```

Comment:
HORRIBLE.  Do not make assumptions on the overall layout dimensions.  Utilize flexbox.